xarray/core/common.py

```diff
@@ -477,6 +477,10 @@ def where(self, cond):
         """
         return self._where(cond)
 
+    __lt__ = __le__ =__ge__ = __gt__ = __add__ = __sub__ = __mul__ = \
```

Comment:
If we can't add tests, let's at least add a comment noting why we added this (otherwise it's likely to do away when someone refactors)